src/app/modules/vault-filter/components/organization-filter.component.html

```diff
@@ -0,0 +1,111 @@
+<ng-container *ngIf="show">
+  <ng-container [ngSwitch]="displayMode">
+    <ng-container *ngSwitchCase="'noOrganizations'">
```

Comment:
I don't think this will ever be shown due to the `show` getter in the component class.
If there are no organizations, `show === false` and the component won't be rendered.

Reply:
[[bug] Remove inaccesible OrganizationFilter template display conditions](https://github.com/bitwarden/desktop/pull/1487/commits/054e5be7e7a1eb03a8b520736cbd64580161d700)